airflow/providers/amazon/aws/exceptions.py

```diff
@@ -42,3 +44,7 @@ def __init__(self, failures: list, message: str):
 
     def __reduce__(self):
         return EcsOperatorError, (self.failures, self.message)
+
+
+class S3HookUriParseFailure(AirflowException):
+    """When parse_s3_url fails to parse URL, this error is thrown."""
```

Comment:
it could, but i chose to do this because it was previously raising airflowexception  ... basically... backcompat...  anything that was previously catching will still catch.  if we change, it won't

Reply:
do you think this is right?

Reply:
OK for compatibility I think it’s fine.